app/components/Feedback/feedback.css

```diff
@@ -1,23 +1,49 @@
-.feedback .composite-button {
-  width: fit-content;
+.feedback {
+  display: flex;
+  align-items: center;
+}
+
+.thanks {
+  transition: opacity 200ms ease-in-out;
+  margin-left: 0.5rem;
+  pointer-events: none;
+}
+.show {
+  opacity: 1;
+}
+
+.hide {
+  opacity: 0;
 }
 
 .feedback-form {
-  position: relative;
-  z-index: 2;
-  max-width: 384px;
-  padding: var(--spacing-32);
-  margin-top: var(--spacing-8);
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: 50px;
+  left: calc(13.333vw + 85px);
+  width: 384px;
+  padding: 30px;
+  border-radius: 6px;
 }
 
 /* Comment to Nemo when he reviews: why didn't you just do this the way you did the settings buttons? */
 .select-option {
   height: var(--spacing-48);
-  padding: var(--spacing-8) var(--spacing-16);
+  border-radius: 6px;
```

Comment:
use a `var` here. Or consider adding the `rounded` class to whatever is using this

Reply:
yes I've changed it. The only reason I used the 6px is because that's what the figma said, but the var for --border-radius is 4px. I totally get the value of only needing to change one variable though so yeah.